test/e2e/antctl_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/stretchr/testify/require"
```

Comment:
could you make sure that `TestAntctlControllerRemoteAccess` tests out-of-cluster `antctl supportbundle`? You may have to add the command to the list manually since it is a "raw command".

Reply:
Done.